spec/pluralrules.html

```diff
@@ -4,39 +4,36 @@
   <emu-clause id="sec-intl-pluralrules-abstracts">
     <h1>Abstract Operations for PluralRules Objects</h1>
 
-    <emu-clause id="sec-InitializePluralRules" aoid="InitializePluralRules">
-      <h1>InitializePluralRules (pluralRules, locales, options)</h1>
+    <emu-clause id="sec-initializepluralrules" aoid="InitializePluralRules">
+      <h1>InitializePluralRules ( _pluralRules_, _locales_, _options_ )</h1>
 
       <p>
-        The abstract operation InitializePluralRules accepts the arguments _pluralRules_ (which must be an object), _locales_, and _options_. It initializes _pluralRules_ as a PluralRules object. It performs the following steps:
+        The abstract operation InitializePluralRules accepts the arguments _pluralRules_ (which must be an object), _locales_, and _options_. It initializes _pluralRules_ as a PluralRules object. The following steps are taken
       </p>
 
       <emu-alg>
-        1. If _pluralRules_.[[InitializedIntlObject]] is *true*, throw a *TypeError* exception.
-        1. Set _pluralRules_.[[InitializedIntlObject]] to *true*.
         1. Let _requestedLocales_ be ? CanonicalizeLocaleList(_locales_).
         1. If _options_ is *undefined*, then
-          1. Let _options_ be ObjectCreate(*%ObjectPrototype%*).
```

Comment:
Good catch, thanks for the fix. This is an observable change; we should make sure it's covered by test262 tests. It already matches the V8 implementation, as you can see in this test:

`(Object.prototype.type = 'ordinal') && new Intl.PluralRules().resolvedOptions().type  // => "cardinal"`